README.md

```diff
@@ -1 +1,3 @@
-# caf-ssl
\ No newline at end of file
+# caf-ssl
+
+This project provides SSL utility code.
```

Comment:
We should add a bit more here.  Like how to use the module, what it's for, etc.  But we can do that in a separate PR if you want to get this merged.